### ISSUE TITLE
Expand documentation

### ISSUE THREAD

**Issue by cooperq** [EFForg/rayhunter#29]
Document all of the code and expand the readme and wiki 
include docs on how to contribute and community guidelines

**Comment by Grewfisk** [comment]
Here's instructions on installing with an Ubuntu 24.04 Live USB

### Create a USB Installer for Ubuntu
1. Download Ubuntu Desktop 24.04 from [here](https://ubuntu.com/download/desktop)
2. Write to a USB stick of at least 8GB in size. On windows you can use Rufus. On gnome based linux systems you can use Disks to "Restore image" onto the USB stick.

### Setup Live USB for use

1. Boot from the USB stick. This will require you to modify the boot order in your BIOS or press the key combination for the boot menu on your computer. Each computer is different. Refer to your manual.
2. Once booted you'll be presented with options. Select "Try or Install" and wait for it to start.
3. Once Ubuntu has started the installer window will open. Press X in the upper right corner to dismiss it.
4. Click the control bar in the upper right corner to open the menu to connect to the internet. Connect to a wifi access point or verify that your wired connection is active.
5. Open Firefox and go to the Rayhunter releases page and download the release.
6. Open the download folder, right click `release.tar` and choose "Extract"
7. Open the release folder. Right click and select "Open in Terminal"
8. Install curl by typing `sudo apt install curl` then hit enter. Confirm your choice by typing Y and hitting enter.

### Install Rayhunter

1. Connect the device to the computer's USB port
2. In the same Terminal type `sudo sh install-linux.sh` and then hit enter.
3. Rayhunter will be installed after several steps. Wait for it to complete.

**Comment by Grewfisk** [comment]
Add a note that verizon branded devices are unlocked from the factory and will work with any SIM

**Comment by untitaker** [comment]
i don't think this is relevant anymore -- docs generally grew by a lot, contributing guidelines kind of do exist, and the installer is sufficiently cross-platform now that one shouldn't have to have a ubuntu live cd for best results anymore

what's missing is code documentation but that's always the case to some degree in any codebase...
